app/serializers/api/admin/csv/goods_nomenclature_serializer.rb

```diff
@@ -10,17 +10,19 @@ class GoodsNomenclatureSerializer
         column :description, column_name: 'Description'
         column :validity_start_date, column_name: 'Start date'
         column :validity_end_date, column_name: 'End date'
-        column :number_indents, column_name: 'Indentation'
-        column :end_line, column_name: 'End line', &:path_declarable?
+
+        column(:number_indents, column_name: 'Indentation') do |goods_nomenclature|
```

Comment:
I think you could add this to the goods_nomenclature model as ns_number_indents/equivalent?